gatsby-config.js

```diff
@@ -4,7 +4,7 @@ require('dotenv').config()
 const path = require('path')
 
 require('./config/prismjs/dvc')
-require('./config/prismjs/usage')
+// require('./config/prismjs/usage')
```

Comment:
does this need deleting?

Reply:
Yes, because we are using custom component for Code block and the logic on `prismjs/usage` are also migrated to that custom Code component. 
We needed parsing the code block and it was relative easy to work with raw data than complex structure provided after allowing through prismjs.

Reply:
we could completely remove this prism plugin then? let's do this then